src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx

```diff
@@ -3,14 +3,20 @@ import classNames from "classnames";
 
 import "./AdminShowcase.scss";
 
-export interface AdminPanelProps {
+export interface AdminShowcaseProps {
   className?: string;
+  variant?: "internal-scroll" | "no-scroll";
```

Comment:
`.AdminShowcase--no-scroll` hasn't been created. I'm assuming that's intentional and it's just to make it clear to anyone looking at this what is going on.

Reply:
We've usually been adding [NOTE] and [OOS] as a shorthand for things that probably shouldn't be acted upon on the particular PR

Reply:
Right now it's intentional since  the `variant` system is still new. Some of those values should probably converge in the future, once we have more use cases and we determine similarities/differences. Also, the `?` is there to facilitate legacy uses and generally should be avoided (rectified in future PR)